testdata/import-dot.go

```diff
@@ -0,0 +1,18 @@
+// Test that dot imports are flagged.
+
+// Package pkg ...
+package pkg
+
+import (
+	. "context" // in allowedPackages (standard library => just the name without full path)
```

Comment:
Can we please have a non-aliased and aliased imports to make sure they are not affected.

Reply:
Thanks for the review @denisvmedia. Fixed. PTAL.